internal/dbusservice/dbusservice.go

```diff
@@ -119,7 +121,16 @@ func New(_ context.Context, cfgPath, cachePath string) (s *Service, err error) {
 
 // parseConfig parses the config file and returns a map with the configuration keys and values.
 func parseConfig(cfgPath string) (map[string]map[string]string, error) {
-	iniCfg, err := ini.Load(cfgPath)
+	cfgContent, err := os.ReadFile(cfgPath)
+	if err != nil {
+		return nil, err
+	}
+
+	if string(cfgContent) == conf.BrokerConfTemplate {
```

Comment:
I am unsure about that change. Do you feel this is more interesting to the user than knowing they are missing X or Y fields to be valid?

Let me give you an example when this approache fails:
* you install one broker, don’t configure it
* the broker refreshes and we changed the config (added a comment)
* some days after the user starts the broker: this error won’t be triggered.

I really think a better approach is really to valid the fields we need to be edited (maybe not containing any template markup?)

Reply:
Agreed, this is a better approach!